site/public/templates/grading/OverallComment.twig

```diff
@@ -2,35 +2,52 @@
 Required Parameters:
 -grading_disabled: if the the element should be inactive
 -editable: if the element is in edit mode
--overall_comment: the text to show in box
+-overall_comment: array with the following keys/values:
```

Comment:
I'm getting the following error when trying to view my grade, as a student. It seems like the recent fix that changed overall comment in the TA view to a multi-tab system has not been accounted for in this twig.

Reply:
Pushed a fix